app/helpers/application_helper.rb

```diff
@@ -337,21 +337,11 @@ def impersonating?
   end
 
   def saas?
```

Comment:
I've seen you added `saas?` method to `deploy.rb` after changing this one so, maybe we don't need this?